# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/HTMLEditor.h
# Commit: d61ac9d43588
# Full Hash: d61ac9d4358882e3b27761a6344eb9ad0e13466c
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-08-04 04:34:13
# Regressor Bug: 1774704
# File Overlap Count: 1
# Description:
#   Bug 1774704 - part 4-1: Move `HTMLEditor::GetParentListElementAtSelection` into `AutoRangeArray` r=m_kato
#   
#   It oddly retrieve an ancestor list element which contains one range in the
#   selection ranges.  So, working it with `AutoRangeArray` which is initialized
#   with `Selection` makes `HTMLEditor` smaller...
# ==============================================================================

diff -r 0b2613e3fdfe -r d61ac9d43588 editor/libeditor/HTMLEditor.h
--- a/editor/libeditor/HTMLEditor.h	Thu Aug 04 04:45:47 2022 +0300
+++ b/editor/libeditor/HTMLEditor.h	Thu Aug 04 01:50:28 2022 +0000
@@ -1230,13 +1230,6 @@
   }
 
   /**
-   * Try to get parent list element at `Selection`.  This returns first find
-   * parent list element of common ancestor of ranges (looking for it from
-   * first range to last range).
-   */
-  Element* GetParentListElementAtSelection() const;
-
-  /**
    * GetRangeExtendedToHardLineEdgesForBlockEditAction() returns an extended
    * range if aRange should be extended before handling a block level editing.
    * If aRange start and/or end point <br> or something non-editable point, they
